tuna/example/example_tables.py

```diff
@@ -41,19 +43,21 @@ class JobEnum(enum.Enum):
   """Represents job_enum column in config table"""
   # pylint: disable=invalid-name ; names represent entries in job_enum column
   # pylint: disable=duplicate-code
-  new = 1
-  running = 3
-  completed = 4
-  error = 5
+  new: int = 1
+  running: int = 3
+  completed: int = 4
+  error: int = 5
```

Comment:
Definition for an Enum would by necessity be an int. If mypy doesn't complain, I don't think we need the int hints here.